plugin/docker/container.go

```diff
@@ -88,6 +88,14 @@ func (c *container) List(ctx context.Context) ([]plugin.Entry, error) {
 	return []plugin.Entry{clf, cm, vol.NewFS("fs", c, 3)}, nil
 }
 
+func (c *container) Delete(ctx context.Context) (bool, error) {
+	err := c.client.ContainerRemove(ctx, c.id, types.ContainerRemoveOptions{
+		// TODO: Should we set RemoveVolumes and RemoveLinks too?
```

Comment:
I'm not sure I understand what these mean. I'd guess we don't want to. I think those are things that can persist after and be reused, so they should be removed separately.

Reply:
I noticed those options in `docker rm` so thought I'd ask. I'll remove the TODO.